common/big_messaging/src/test/scala/uk/ac/wellcome/bigmessaging/message/MessagingFixtureIntegrationTest.scala

```diff
@@ -91,29 +98,59 @@ class MessagingIntegrationTest
                         BigMessageSender[SNSConfig, ExampleObject]),
                        R]): R = {
     withLocalStackMessageStreamFixtures[R] {
-      case (queue, messageStream) =>
+      case (queue, messageStream, store) =>
         withLocalS3Bucket { bucket =>
           withLocalStackSnsTopic { topic =>
             withLocalStackSubscription(queue, topic) { _ =>
-              withSqsBigMessageSender[ExampleObject, R](
-                bucket,
-                topic,
-                localStackSnsClient) { messageWriter =>
-                testWith((messageStream, messageWriter))
+              withBigMessageSender(bucket, topic, localStackSnsClient, store) {
+                messageWriter =>
+                  testWith((messageStream, messageWriter))
               }
             }
           }
         }
     }
   }
 
+  def withBigMessageSender[R](
+    bucket: Bucket,
+    topic: Topic,
+    senderSnsClient: AmazonSNS = snsClient,
+    store: MemoryTypedStore[ObjectLocation, ExampleObject])(
+    testWith: TestWith[BigMessageSender[SNSConfig, ExampleObject], R])(
+    implicit
+    circeEncoder: Encoder[ExampleObject]
+  ): R = {
```

Comment:
Isn't the code in here similar to the fixture code in `BigMessagingFixture`? Maybe could generalise the method there and use it here, rather than repeating here if so.